test/runtests.jl

```diff
@@ -0,0 +1,5 @@
+using OceanEnsembles, Test
+
+a = 1
+
+@test a == 1
```

Comment:
ideally we want to include here tests for the methods introduced by OceanEnsembles; but we can leave it as a placeholder for this PR